### DISCUSSION TITLE
Welcome to plex-media-server-stack Discussions!

### DISCUSSION THREAD

**Discussion by joshdev8** [joshdev8/AutoPlexx#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Taksh13** [comment]
Hey Man, kinda confused on how to get it started. I am on Ubuntu, i have docker and portainer installed. The thing that throwing me off is the env file, it mentions the keys for Radarr, Sonarr, etc but to get the keys i need to set them up in docker right? I guess I need some info on the order of the steps. Thanks in advance

**Reply by joshdev8** [reply]
Hey @Taksh13 — great question, and you've caught something real about that env file.

The `RADARR_API_KEY`, `SONARR_API_KEY`, and several other vars in `.env.example` are vestigial — nothing in the current `docker-compose.yml` actually reads them. You can ignore them. They're being removed in an upcoming cleanup PR.

**The real startup order:**

1. `cp .env.example .env`
2. Fill in the values that actually matter:
   - `PUID`, `PGID`, `TZ`, `USERDIR` — used by every service
   - `DB_PASSWORD`, `JWT_SECRET`, `COOKIE_SECRET` — Tracearr won't start without them; any strong random strings work (e.g. `openssl rand -base64 32`)
   - `PMM_*` — only if you want to run Kometa for collections/overlays
3. **Right before step 4**, grab a Plex claim token from https://www.plex.tv/claim and paste it into `PLEX_CLAIM`. Claim tokens expire in ~4 minutes, so do this last.
4. `docker compose up -d`
5. Once the stack is running, hit the UIs (Radarr `:7878`, Sonarr `:8989`, Tautulli `:8181`, etc.) to configure libraries and pull API keys from there if you want them for integrations.

Since you already have Portainer installed: the latest compose file now ships a Portainer container on port 9000. You can either use the one in the stack (and remove yours) or comment out the `portainer` block in `docker-compose.yml` to keep your existing setup.

If you get stuck on a specific step let me know which one and I can dig in.

**Reply by joonboi** [reply]
this order of operations is good, but i have a question. what if i already have a plex server claimed and want to transfer it to this.  (just to get everything else around it working) how would i do that?

**Reply by joshdev8** [reply]
@joonboi try removing or just commenting out the Plex block in the docker-compose.yml file, and run it. Let me know if you run into any issues.
